internal/namespaces/k8s/v1beta4/custom_kubeconfig_uninstall.go

```diff
@@ -0,0 +1,126 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/scaleway/scaleway-cli/internal/core"
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1beta4"
+	"gopkg.in/yaml.v2"
+)
+
+type k8sKubeconfigUninstallRequest struct {
+	ClusterID string
+}
+
+func k8sKubeconfigUninstallCommand() *core.Command {
+	return &core.Command{
+		Short:     `Uninstall a kubeconfig`,
+		Long:      `Remove specified cluster from kubeconfig file specified by the KUBECONFIG env, if empty it will default to $HOME/.kube/config.`,
+		Namespace: "k8s",
+		Verb:      "uninstall",
+		Resource:  "kubeconfig",
+		ArgsType:  reflect.TypeOf(k8sKubeconfigUninstallRequest{}),
+		ArgSpecs: core.ArgSpecs{
+			{
+				Name:       "cluster-id",
+				Short:      "Cluster ID from which to uninstall the kubeconfig",
+				Required:   true,
+				Positional: true,
+			},
+		},
+		Run: k8sKubeconfigUninstallRun,
+	}
+}
+
+func k8sKubeconfigUninstallRun(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+	request := argsI.(*k8sKubeconfigUninstallRequest)
+
+	// get the path to write the wanted kubeconfig on disk
+	// either the file pointed by the KUBECONFIG env variable (first one in case of a list)
+	// or the $HOME/.kube/config
+	var kubeconfigPath string
+	kubeconfigEnv := core.ExtractEnv(ctx, "KUBECONFIG")
+	if kubeconfigEnv != "" {
+		if runtime.GOOS == "windows" {
+			kubeconfigPath = strings.Split(kubeconfigEnv, ";")[0]
+		} else {
+			kubeconfigPath = strings.Split(kubeconfigEnv, ":")[0]
+		}
+	} else {
+		homeDir, err := homedir.Dir()
+		if err != nil {
+			return nil, err
+		}
+		kubeconfigPath = path.Join(homeDir, kubeLocationDir, "config")
+	}
+
+	// if the file does not exist, the cluster is not there
+	if _, err := os.Stat(kubeconfigPath); os.IsNotExist(err) {
+		return fmt.Sprintf("File %s does not exists.", kubeconfigPath), nil
+	}
+
+	// getting the existing file
+	file, err := ioutil.ReadFile(kubeconfigPath)
```

Comment:
is this a duplicate from `install` ? 

can we use a util function instead of code duplication ?